src/app/panels/terminal.js

```diff
@@ -105,23 +105,40 @@ class Terminal {
         ${self._view.input}
       </div>
     `
-    self._view.icon = yo`<i onmouseenter=${hover} onmouseleave=${hover} onmousedown=${minimize} class="${css.toggleTerminal} fa fa-angle-double-down"></i>`
-    self._view.dragbar = yo`<div onmousedown=${mousedown} class=${css.dragbarHorizontal}></div>`
+    self._view.icon = yo`
+      <i onmouseenter=${hover} onmouseleave=${hover} onmousedown=${minimize}
```

Comment:
we might miss a title for the new `pending transaction` div, or we could display the entire `x pending transactions` ....

Reply:
shall we add this label here to clarify what the counter means?

Reply:
Absolutely. It must have disappeared during rebasing.